tests/cross/src/test/scala/tests/pc/SignatureHelpSuite.scala

```diff
@@ -925,4 +925,144 @@ class SignatureHelpSuite extends BaseSignatureHelpSuite {
        |""".stripMargin,
   )
 
+  // NOTE:
+  // Unignore after merging https://github.com/lampepfl/dotty/pull/18594
```

Comment:
This has been already merged a while ago, so maybe the test could be enabled now. :slightly_smiling_face: